docs/content/testing/testing-guidelines.md

```diff
@@ -0,0 +1,238 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Testing guidelines
+
+The tests should:
+
+- Set up some initial state
+- Execute a method that acts upon this state and results in a new state
+- Make assertions in the new state against the expected changes
+- Clean up
+
+NUnit allows this to be done through the use of fixtures and special method annotations.
+
+## Module directory structure
+
+The Unity GDK project's assets are divided into modules, such as `Core`, `Physics`, `PlayerLifecycle`.
+
+Each module is in its own directory within `workers\unity\Assets\Gdk`.
+
+Each module has its own tests that cover the functionality of that module.
+
+Unity Test Runner also allows separate Playmode and Editmode tests. For more information on this division, please refer to the [Unity Test Runner manual page](https://docs.unity3d.com/Manual/testing-editortestsrunner.html).
+
+Within the root of the module, the `Tests` directory should contain `Editmode`
+ and `Playmode` subdirectories with their own assembly definition files.
+
+Test assembly definition files must match this pattern:
+> `Improbable.Gdk.<ModuleName>.<TestMode>Tests.asmdef`
+
+### Editmode vs. Playmode tests
```

Comment:
"Edit mode" and "Play mode"?

Reply:
Unity calls them `Editmode` and `Playmode` so I want to keep consistent with them

Reply:
Correction: `EditMode` / `PlayMode`